### PR TITLE
graphite_exporter: update 0.2.0->0.7.1

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Review by foxxx0 — APPROVED** [review]

**Comment by dhoppe** [comment]
Basically this pull request works, but you should increase the version of graphite_exporter to 0.7.1.

**Review by dhoppe — APPROVED** [review]

**Comment by ghoneycutt** [comment]
@dhoppe From your comment It is not clear if this is ready for merge. Does it need a change made for 0.7.1 beforehand? If so, could you add it to this PR?

File: spec/acceptance/graphite_exporter_spec.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    describe service('graphite_exporter') do
+      it { is_expected.to be_running }
+      it { is_expected.to be_enabled }
+    end
+
+    describe port(9109) do
+      it { is_expected.to be_listening.with('tcp6') }
+    end
+    it 'is idempotent' do
+      pp = "class{'prometheus::graphite_exporter': version => '0.6.2'}"
```
Comment (ghoneycutt) [inline spec/acceptance/graphite_exporter_spec.rb]: If you are going to specify the version, you should check that the specified version is what got installed.

**Review by ghoneycutt — CHANGES_REQUESTED** [review]
acceptance testing should validate version

File: spec/acceptance/graphite_exporter_spec.rb (hunk 1 above)
Comment (dhoppe) [reply spec/acceptance/graphite_exporter_spec.rb]: I am not the author and this branch does not belong to my namespace.

**Comment by bastelfreak** [comment]
we already have tests inplace that ensure the module workflow works as expected. I don't think that a test is required to explicitly validate the version. I'm already busy enough updating the exporter versions. Feel free to submit patches if you like to see such tests in our acceptance test suite.

**Pull request merged**